### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by IodaMikeMapbox**
Checks:

* [ ] Update changelog (not required)
* [x] Rebase to `dev` branch
* [x] Assign reviewers

Linked issues:
Part of https://github.com/mapbox/mapbox-vision/issues/1748

### CODE REVIEW

File: Examples/VisionExamples/Examples/ARNavigationViewController.swift [hunk 1]
```diff
@@ -28,7 +28,7 @@ class ARNavigationViewController: UIViewController {
         visionManager = VisionManager.create(videoSource: videoSource)
         // create VisionARManager and register as its delegate to receive AR related events
         visionARManager = VisionARManager.create(visionManager: visionManager)
-
+        /// configure ar view to display AR navigation
```
Comment (chezzdev) [inline Examples/VisionExamples/Examples/ARNavigationViewController.swift]:
```suggestion
        /// configure AR view controller to display AR navigation
```

Comment (IodaMikeMapbox) [reply Examples/VisionExamples/Examples/ARNavigationViewController.swift]: done

**Comment by vladzima** [comment]
Issue?

**Comment by chezzdev** [comment]
This is a part of mapbox/mapbox-vision#1748

File: Examples/VisionExamples/Examples/ARNavigationViewController.swift [hunk 2]
```diff
@@ -26,9 +26,9 @@ class ARNavigationViewController: UIViewController {
 
         // create VisionManager with video source
         visionManager = VisionManager.create(videoSource: videoSource)
-        // create VisionARManager and register as its delegate to receive AR related events
+        // create VisionARManager
         visionARManager = VisionARManager.create(visionManager: visionManager)
-
+        /// configure AR view to display AR navigation
```
Comment (dersim-davaod) [inline Examples/VisionExamples/Examples/ARNavigationViewController.swift]: @IodaMikeMapbox do we want to use a doc comment or this is a typo?

Comment (IodaMikeMapbox) [reply Examples/VisionExamples/Examples/ARNavigationViewController.swift]: ah, right, it's a typo

**Pull request merged**